db/schema.rb

```diff
@@ -48,9 +48,9 @@
     t.datetime "pledge_generated_at", precision: nil
     t.datetime "pledge_sent_at", precision: nil
     t.boolean "textable"
-    t.bigint "clinic_id"
-    t.bigint "pledge_generated_by_id"
-    t.bigint "pledge_sent_by_id"
+    t.integer "clinic_id"
```

Comment:
these have gone back and forth several times and I'm choosing not to think about it very hard anymore